### ISSUE TITLE
scrolling to the top

### ISSUE THREAD

**Issue by mahouha** [oumoussa98/vue3-infinite-loading#39]
Hi, I'm trying to use this helpful package fro my chat app but the documentation was not very detailed,  <br>
so I used it like this `<InfiniteLoading @infinite="load" target="#div" :top="true" />` to scroll the div from bottom to top but it not works

**Comment by oumoussa98** [comment]
Hi @mahouha, I am unable to reproduce this
If you manage to find a repro case, would help a lot.
However, here is a [codepen example](https://codepen.io/oumoussa98/full/vYjZQRO) that might help.

**Comment by tadvas** [comment]
Hi there,
facing issues with props as well
Target doesn't work :(
`<InfiniteLoading @infinite="load" target="#lds-ripple"/> `

I am playing with 1.2.1 version

**Comment by oumoussa98** [comment]
Hi @tadvas, The target prop will default to the `viewport` if `#lds-ripple` doesn't exist.
If you manage to find a repro case, we will gladly reopen this issue.

**Comment by AngryChocobo** [comment]
> Hi @mahouha, I am unable to reproduce this If you manage to find a repro case, would help a lot. However, here is a [codepen example](https://codepen.io/oumoussa98/full/vYjZQRO) that might help.

TLDR 
Scroll to top you must know: 
1. in template put `<InfiniteLoading />` before `<ul />` 
2.  `<InfiniteLoading />` must have `top` property
3. update data must use `unshift` instead of `push`
